ui-v2/src/components/task-runs/task-run-details/task-run-details.tsx

```diff
@@ -128,6 +137,19 @@ export const TaskRunDetails = ({ taskRun }: TaskRunDetailsProps) => {
 				<dd className="font-mono ">{taskRun.id}</dd>
 			</dl>
 
+			{resultArtifact?.description && (
+				<dl className="flex flex-col gap-1 mb-2">
+					<dt className="text-gray-500">Result</dt>
```

Comment:
Update this to use a class that updates depending on if the app is in light mode or dark mode.